unixfs.ts

```diff
@@ -0,0 +1,481 @@
+/**
+ * Logical representation of a file chunk.
+ * 
+ * TODO: Clarify when this represenation is used as opposed to `FileChunk`.
```

Comment:
This is the `leafType` option of the importer and I believe unused unless you specify it. If you specify `raw` for `leafType` you'll get these type of nodes at the leaves of the DAG.

https://github.com/ipfs/js-ipfs-unixfs/tree/master/packages/ipfs-unixfs-importer#api

Reply:
I'm aware of this, although I don't see corresponding option in `ipfs` CLI and generally don't understand pros/cons between choosing file vs raw for chunks.

Is one of them legacy and mostly for backwards compat ? Or is there legitimate reason to have both and one should / would choose one over the other ?